src/sentry/options/defaults.py

```diff
@@ -348,6 +348,14 @@
     type=Int,
     flags=FLAG_AUTOMATOR_MODIFIABLE,
 )
+register(
+    "deletions.group-hash-metadata.use-old-update-method",
+    default=False,  # Default using new update method
```

Comment:
We go live with the new code right away and then use an option if we have to.
This reduces the unnecessary options dance.

Reply:
do you already have a PR up in case we need to turn it off? using the `default=True` method means we can always quick revert in case which we don't have the option to do here

Reply:
I will have it ready